video-intelligence-label-detection/functions/src/index.ts

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as functions from "firebase-functions";
+import * as videoIntelligence from "@google-cloud/video-intelligence";
+import config from "./config";
+import * as path from "path";
+
+const { logger } = require("firebase-functions");
+const { Feature } = videoIntelligence.protos.google.cloud.videointelligence.v1;
+
+const admin = require("firebase-admin");
+admin.initializeApp();
```

Comment:
```suggestion
```

We don't make use of the Admin SDK here so we can remove it and also from `package.json`